xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/VideoFilterShader.h

```diff
@@ -57,11 +57,41 @@ namespace Shaders {
     GLint m_hStretch = 0;
   };
 
+  class BaseVideoFilterShaderGLES : public BaseVideoFilterShaderGL
```

Comment:
this is not a good idea. I consider OpenGL reference and don't want ports like GLES to break when the reference changes.